packages/@aws-cdk/aws-appconfig-alpha/test/configuration.test.ts

```diff
@@ -149,6 +149,45 @@ describe('configuration', () => {
     Template.fromStack(stack).resourceCountIs('AWS::AppConfig::Deployment', 1);
   });
 
+  test('configuration using deploy method', () => {
```

Comment:
Can you please add unit tests for:
* Configuration with no env and no deployTo that calls method
* Configuration with env and deployTo that calls method